clientv3/integration/lease_test.go

```diff
@@ -299,7 +299,7 @@ func TestLeaseGrantErrConnClosed(t *testing.T) {
 	}
 
 	select {
-	case <-time.After(3 * time.Second):
+	case <-time.After(leaseRequestTimeout):
```

Comment:
probably rename this to requestWaitTimeout? also comment this const as `time duration to wait for a request to go through`